src/test/scala/org/kibanaLoadTest/helpers/CloudHttpClient.scala

```diff
@@ -26,21 +27,21 @@ class CloudHttpClient {
     .get
   val logger: Logger = LoggerFactory.getLogger("httpClient")
 
-  def preparePayload(config: Config): String = {
+  def preparePayload(stackVersion: String, config: Config): String = {
     logger.info(
       s"preparePayload: Using ${deployPayloadTemplate} payload template"
     )
     val template = Helper.loadJsonString(deployPayloadTemplate)
-    logger.info(s"preparePayload: Using ${config.toString}")
+    logger.info(
+      s"preparePayload: Stack version ${stackVersion} with ${config.toString} configuration"
+    )
 
     val payload = template
       .update('name ! set[String](s"load-testing-${Instant.now.toEpochMilli}"))
```

Comment:
Is this string concatenation?  Perhaps it's adding a string to set of strings?

Reply:
`s"load-testing-${Instant.now.toEpochMilli}"` is string concatenation, but the whole expression updates json string parameter `name` with a new string value